packages/synapse-react-client/vite.config.ts

```diff
@@ -20,6 +21,11 @@ const config = new ConfigBuilder()
       // Do not clean the output directory before building, since we build ESM/CJS and UMD separately.
       emptyOutDir: false,
     },
+    test: {
+      // require vitest tests to be .vitest.ts(x) while we migrate from Jest
```

Comment:
Great approach to moving to vitest over time, and ambitious to start with the old MarkdownSynapse component tests!